docs/notebooks/MPSettingsEditor.ipynb

```diff
@@ -0,0 +1,723 @@
+    "   - Browse - use the controls to navigate to find config.json\n",
+    "   - Search - set the starting directory to search and open the **Search** drop-down\n",
+    "   - When you see the file click on it and click **Select File** button (below the file browser)\n",
+    "   - optionally, click **View Settings** to confirm that this looks right\n",
+    "3. Convert to convert to msticpyconfig format\n",
+    "   - click **View Settings**\n",
+    "4. Save the file\n",
+    "   - type a path into the **Current file** text box\n",
+    "   - Click on **Save file**\n",
+    "5. You can set this file to always load by assigning the path to an environment variable.\n",
+    "   See [Setting the path to your msticpyconfig.yaml](#Setting-the-path-to-your-msticpyconfig.yaml)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mpconfig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### This is the equivalent from the command line"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mpconfig = MpConfigFile()\n",
+    "mpconfig.load_from_file(\"E:\\\\src\\\\asi-jupyter\\\\notebooks\\\\config.json\")\n",
+    "mpconfig.map_json_to_mp_ws()\n",
+    "mpconfig.settings\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Edit your msticpyconfig settings\n",
+    "\n",
+    "## Azure Sentinel Workspaces\n",
+    "\n",
+    "If you loaded a config.json file you should see your workspace displayed. If not,\n",
+    "you can add one or more workspaces here. The Name, WorkspaceId and TenantId are\n",
+    "mandatory. The other fields are helpful but not essential.\n",
+    "\n",
+    "Use the Help drop-down panel to find more information about adding workspaces and finding\n",
+    "the correct values for your workspace.\n",
+    "\n",
+    "If this the workspace that you use frequently or all of the time, you may want to set this as the default.\n",
+    "This creates a duplicate entry named \"Default\" and this is used when you connect to AzureSentinel as\n",
+    "the default workspace to connect to (you can override this by specifying a workspace name at connect time).\n",
+    "\n",
+    "When you've finished, type a file name (usually \"msticpyconfig.yaml\") into the **Conf File** text box\n",
+    "and click **Save File**,\n",
+    "\n",
+    "You can also try the **Validate Settings** button. This should show that you have a few missing\n",
+    "sections (we'll fill these in later) but should show nothing under the the \"Type Validation Results\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mpedit2 = MpConfigEdit(settings={})\n",
+    "mpedit2"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Why the two cells here?

Reply:
Removed the offending experimental cell.